airbyte-integrations/connectors/source-orb/integration_tests/catalog.json

```diff
@@ -0,0 +1,144 @@
+            "billing_address": {
+              "type": ["null", "object"],
+              "properties": {
+                "city": { "type": ["null", "string"] },
+                "country": { "type": ["null", "string"] },
+                "line1": { "type": ["null", "string"] },
+                "line2": { "type": ["null", "string"] },
+                "postal_code": { "type": ["null", "string"] },
+                "state": { "type": ["null", "string"] }
+              }
+            }
+          }
+        },
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["created_at"],
+        "source_defined_primary_key": [["id"]]
+      },
+      {
+        "name": "subscriptions",
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": ["null", "object"],
+          "properties": {
+            "id": { "type": "string" },
+            "created_at": { "type": ["null", "string"], "format": "date-time" },
+            "customer": {
+              "type": "object",
+              "properties": { "id": { "type": "string" } }
+            },
+            "start_date": { "type": ["null", "string"], "format": "date-time" },
+            "end_date": { "type": ["null", "string"], "format": "date-time" },
+            "plan": {
+              "type": "object",
+              "properties": { "id": { "type": "string" } }
+            },
+            "status": { "type": "string" }
+          }
+        },
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["created_at"],
+        "source_defined_primary_key": [["id"]]
+      },
+      {
+        "name": "plans",
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": ["null", "object"],
+          "properties": {
+            "id": { "type": "string" },
+            "created_at": { "type": ["null", "string"], "format": "date-time" },
+            "description": { "type": ["null", "string"] },
+            "name": { "type": ["null", "string"] },
+            "prices": {
+              "type": ["array"],
+              "items": {
+                "type": "object",
+                "properties": { "id": { "type": "string" } }
+              }
+            },
+            "product": {
+              "type": "object",
+              "properties": { "id": { "type": "string" } }
+            },
+            "start_date": { "type": ["null", "string"], "format": "date-time" },
+            "end_date": { "type": ["null", "string"], "format": "date-time" }
+          }
+        },
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["created_at"],
+        "source_defined_primary_key": [["id"]]
+      },
+      {
+        "name": "credits_ledger_entries",
```

Comment:
@kgrover would it be possible to include the usage event properties that we sent to Orb in the credit ledger entry output? This would give us more flexibility to analyze the usage data, e.g. being able to group the output on event properties, or use the event properties to join this data against other tables

Reply:
@lmossman thanks for bringing this up - let me chat with the team and get back to you on this request!